src/main/java/server/EchoServerWrappers.java

```diff
@@ -0,0 +1,58 @@
+package server;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.net.ServerSocket;
+import java.net.Socket;
+
+public class EchoServerWrappers implements EchoServerInterfaces{
```

Comment:
Before merging, take another look at formatting throughout your files -- like, there should be a space before the open curly brackets on this line.  And between methods there should be one empty line.  Sometimes I see two, sometimes I see none.

Reply:
spacing has been resolved